pdns/recursordist/docs/changelog/4.2.rst

```diff
@@ -0,0 +1,10 @@
+Changelogs for 4.2.x
+====================
+
+.. changelog::
+  :version: 4.2.0-alpha1
+  :released: 1st of February 2019
+
+  Initial 4.2.x release, so no changelog.
```

Comment:
Please link to the blogpost or copy the new features and deprecations in here :)

Reply:
I'll link it to the blogpost but I only know the link once we've published it :-)